docs/src/modules/concepts/pages/concepts.adoc

```diff
@@ -114,13 +114,43 @@ Timers:: Time-based triggers
 
-Akka components can communicate seamlessly, regardless of physical location. Location transparency ensures efficient message routing and runtime efficiency, even across network boundaries.
+Akka components are able to interoperate with one another regardless of where the component may be executing. While you create systems that define the relationships between each of the components in code, at runtime instances of each component can run on different nodes. The Akka runtime provides location transparency, automating the routing of a call from one component to another even if they reside in different locations on different networks.
 
-There are two main modes of inter-component communication in Akka:
+There are two ways to achieve interoperability between Akka components and the outside world.
 
-1. **ComponentClient**: Asynchronous, non-blocking invocations. Components return immediately after dispatching a message, regardless of destination.
 
-2. **Events**: Components emit events that others may subscribe to. This pattern enables decoupled, brokerless messaging similar to pub/sub systems. Akka components may also subscribe to events from external sources (e.g., APIs, brokers).
+[cols="1,3", options="header"]
+|===
+|Client Type |Description
+
+|xref:java:component-and-service-calls.adoc[ComponentClient]
+|One component can directly invoke another component. Akka treats these direct invocations as non-blocking, asynchronous messages, meaning that the invoking component immediately returns after making the invocation and can invoke any component regardless of its location.
+
+For example, a Workflow component handling a long-running business transaction can invoke an Agent component to perform a task, and the Workflow will continue processing without waiting for the Agent to complete its work.
+
+|Events
+|Components can emit events that can be subscribed to by other components. For example, an entity can emit an event when its state is updated. Other components can subscribe to this event, similar to how clients subscribe to Kafka topics. Events are propagated transparently between components through reliable, brokerless messaging.
+
+Components can also subscribe to events that are incoming from external sources, which can be 3rd party brokers, APIs, or real-time streams of data.
+|===
+
+Due to this structure, even though the code you create which connects different components together, all communications between these components are decoupled from one another with non-blocking, asynchronous communication. This enables instances of your components to run anywhere with the most efficient use of compute, memory, and I/O while the Akka runtime takes responsibility for routing requests to the location that needs to consume them. This basic behavior is one reason why Akka services, whether stateful or stateless, can scale to 10M TPS.
+
+[cols="2,3", options="header"]
+|===
+| Example Interoperability | Description
```

Comment:
The use cases here doesn't seem to be connect. 

I'm referring to the `+` part. 

For me, it should read like: 

**Endpoint → Workflow → Agent**

A user triggers an HTTP request to start a long-running Workflow. The Workflow manages file processing and passes relevant data or control to an Agent, which will later use it for answering questions.

**Endpoint → Entity → View**

Another Endpoint saves user interaction history into an Entity. A View component then reads from that Entity to reconstruct the user’s conversation history for display or retrieval.

Reply:
I see, but why are we talking about them together? It's not that they use cases a related.

Reply:
In the “Component Interoperability” section of the document referenced in https://github.com/lightbend/kalix/issues/13639,  I believe Tyler’s intent in discussing these components together is to illustrate that they can be deployed as a single service.